lib/originate.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/python3
+import subprocess, sys, os
+TEMPLATE = os.environ['TEZOS_CLIENT'] + " originate contract {0} transferring 0 from bootstrap1 running 'parameter {1} ; storage unit ; code {{ DROP ; UNIT ; NIL operation ; PAIR }}' --burn-cap 10000000 --force"
```

Comment:
If you can delay the mockup base-dir creation until this point, you might want to try using bootstrap accounts instead of calling `originate` in a loop. See https://tezos.gitlab.io/user/mockup.html#tune-mockup-parameters.

Reply:
Ah, brilliant. That would be much cleaner. I'll add that in a future PR.